q2cli/tools.py

```diff
@@ -55,6 +70,9 @@ def export_data(path, output_dir):
               help='The format of the data to be imported. If not provided, '
                    'data must be in the format expected by the semantic type '
                    'provided via --type.')
+@click.option('--show-importable-types', is_flag=True, is_eager=True,
```

Comment:
Can you also expand the `help` string for `--type` to point users to this new option? Something like:

"The semantic type of the artifact that will be created upon importing. Use --show-importable-types to see what importable semantic types are available in the current deployment."